apps/src/music/views/MusicLabView.tsx

```diff
@@ -92,6 +93,11 @@ const MusicLabView: React.FunctionComponent<MusicLabViewProps> = ({
   const validationStateCallout = useAppSelector(
     state => state.lab.validationState.callout
   );
+  const currentPlayheadPosition = getCurrentPlayheadPosition();
```

Comment:
One last nit 🙂 can we grab the current playhead position from redux as well, so that all the data we're using is in sync? Calling the function directly here technically means that this value will be slightly ahead of what's in redux, since it gets dispatched and updated by the playback loop below. It shouldn't actually make a huge difference because the update loop runs pretty frequently, but just to be consistent

Reply:
Good catch.  Done.